rllib/algorithms/algorithm.py

```diff
@@ -826,7 +826,7 @@ def setup(self, config: AlgorithmConfig) -> None:
             self.env_runner_group.sync_env_runner_states(
                 config=self.config,
                 env_steps_sampled=self.metrics.peek(
-                    NUM_ENV_STEPS_SAMPLED_LIFETIME, default=0
+                    (ENV_RUNNER_RESULTS, NUM_ENV_STEPS_SAMPLED_LIFETIME), default=0
```

Comment:
So, we do still sync the metrics across `EnvRunner`s?

Reply:
We do, but only in `setup()` (at very beginning).

Actually, good point. Let me check, whether we properly sync these stats after a worker restart and/or after a checkpoint restore ...